modules/ROOT/pages/upgrade-index.adoc

```diff
@@ -1,35 +1,282 @@
+
+.. Update the Mule deployments running on Runtime Fabric to the latest patch version.
+... Using your web browser, navigate to the *Applications* tab in Runtime Manager.
+... Select an application and select *Manage Application*. 
+... The value in the *Runtime version* field indicates if an update is available. Select the updated version of Mule runtime engine.
+... Select *Deploy* to redeploy the application with the updated Mule runtime engine version.
+... Repeat this for each application running on this Runtime Fabric.
+.. Verify Runtime Fabric is healthy:
+... Using your web browser, navigate to the *Runtime Fabrics* tab in Runtime Manager and verify that Runtime Fabric reports an *Active* status.
+... Open a terminal to a Runtime Fabric controller node.
+... Run `sudo gravity status` and confirm the cluster status is healthy.
+... Keep your terminal open to use in later steps.
+
+.. Get the latest version of the Runtime Fabric installer package:
+... Using your web browser, navigate to Runtime Manager and select the *Runtime Fabrics* tab to view a list of Runtime Fabrics.
+... Select the *Downloads* link.
+... Select the *Copy Link* icon next to *Installer package* to copy the code snippet, as shown in the following example:
++
+```
+rtfctl appliance upgrade --url https://<upgrade-package-information>
+```
+... Paste the code snippet in a text editor for reference. 
+.. Start the upgrade procedure:
+... Using your terminal open on a controller node, run the following command, substituting the value of the <cluster-installer-url> parameter with the *Installer package* URL value: 
++
+----
+sudo ./rtfctl appliance upgrade --url <cluster-installer-url>
+----
++
+This command also supports using the `--file` parameter to reference an installer package already downloaded on the node.
++
+A confirmation is displayed that indicates the upgrade is being performed in the background.
++
+.. Run `sudo gravity status` on a node and verify that the cluster status is “updating”.
+.. Monitor the upgrade procedure progress:
+... Using your terminal on a controller node, run the following command: 
++
+----
+sudo gravity plan
+----
++
+The upgrade can take more than an hour to complete.
++
+The Runtime Fabric and Runtime Manager connection often disconnects multiple times during the upgrade procedure. Connectivity is restored after the upgrade completes.
++
+The upgrade procedure is designed to keep deployed applications running and available to serve requests. The upgrade is applied in a rolling method:
++
+.... A node is selected by the procedure to apply the upgrade.
+.... Applications running on the selected node are redeployed to another node.
+.... The selected node is removed from the cluster.
+.... The upgrade is applied on the selected node.
+.... The selected node is added to the cluster after upgrading.
```

Comment:
Information is much too deeply nested. Try to break up huge tasks with several subtasks into separate subtopics rather than doing this. Here as above, I have copyedited the existing narrative but would really like to see it reworked to avoid this.

Reply:
OOPS...I should have broken this edit up, due to the number of comments and how picky GitHub is about that. 
I am going to attempt to break here....